<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guideWireSimulationClass</class>
 <widget class="QMainWindow" name="guideWireSimulationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>guideWireSimulation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFff">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionSimulationParams"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionQuit"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionCenter"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHowDoI"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFff"/>
   <addaction name="menuSetting"/>
   <addaction name="menuOperation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFileOpen">
   <property name="text">
    <string>Load Vascular</string>
   </property>
  </action>
  <action name="actionSimulationParams">
   <property name="text">
    <string>Simulation Parameters</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start Trainning</string>
   </property>
  </action>
  <action name="actionHowDoI">
   <property name="text">
    <string>How Do I</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About System</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit Trainning</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause Trainning</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="guidewiresimulation.qrc"/>
 </resources>
 <connections/>
</ui>
